toolkit/Makefile

```diff
@@ -45,6 +45,7 @@ RUN_CHECK                            ?= n
 USE_PREVIEW_REPO                     ?= n
 DISABLE_UPSTREAM_REPOS               ?= n
 DISABLE_DEFAULT_REPOS                ?= n
+REPO_SNAPSHOT_TIME                   ?=
```

Comment:
Can we add a `##help:var:...` entry for this? (see above for `TEST_IGNORE_LIST` on line 29 for example).

Reply:
added